x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/rule_details_flyout/tabs/translation/index.tsx

```diff
@@ -109,19 +107,20 @@ export const TranslationTab: React.FC<TranslationTabProps> = React.memo(
             </EuiSplitPanel.Outer>
           </EuiFlexItem>
         </EuiAccordion>
-        {ruleMigration.translation_result === RuleTranslationResult.FULL && (
-          <>
-            <EuiSpacer size="m" />
-            <EuiCallOut
-              color={'primary'}
-              title={i18n.CALLOUT_TRANSLATED_RULE_INFO_TITLE}
-              iconType={'iInCircle'}
-              size={'s'}
-            >
-              {i18n.CALLOUT_TRANSLATED_RULE_INFO_DESCRIPTION}
-            </EuiCallOut>
-          </>
-        )}
+        {ruleMigration.translation_result === RuleTranslationResult.FULL &&
+          !ruleMigration.elastic_rule?.id && (
+            <>
+              <EuiSpacer size="m" />
+              <EuiCallOut
+                color={'primary'}
+                title={i18n.CALLOUT_TRANSLATED_RULE_INFO_TITLE}
+                iconType={'iInCircle'}
+                size={'s'}
+              >
+                {i18n.CALLOUT_TRANSLATED_RULE_INFO_DESCRIPTION}
+              </EuiCallOut>
+            </>
+          )}
```

Comment:
Show "Install rule" callout only when not installed `!ruleMigration.elastic_rule?.id`

Reply:
is this field `ruleMigration.elastic_rule?.id` populated after installation?

Reply:
@logeekal Exactly, when we install the rule we update the `elastic_rule.id` with the actual id of the installed rule (SO). 
Additionally, if a rule matches a predefined rule during the translation, and that predefined rule is already installed, the `elastic_rule.id` is also assigned.

Reply:
Okay. got it.

What happens in case `pre-defined` rule matches but it is not installed?